tutu-frontend/src/app/hakemus/[oid]/perustelu/yleiset/perustelut/components/Lahde.tsx

```diff
@@ -0,0 +1,105 @@
+import { useEffect, useState } from 'react';
+
+import { Stack, useTheme } from '@mui/material';
+import { OphCheckbox, OphTypography } from '@opetushallitus/oph-design-system';
+import { useTranslations } from '@/src/lib/localization/hooks/useTranslations';
+import { Perustelu } from '@/src/lib/types/perustelu';
+
+interface Props {
+  perustelu: Perustelu | undefined;
```

Comment:
tän vois kirjottaa myös näin
```suggestion
  perustelu?: Perustelu;
```